app/controllers/cohorts_controller.rb

```diff
@@ -2,7 +2,33 @@ class CohortsController < ApplicationController
   after_action :verify_authorized, :except => :index
   after_action :verify_policy_scoped, :only => :index
   def show
-    @cohort = Cohort.find(params[:id])
+    @cohort      = Cohort.find(params[:id]).decorate
```

Comment:
Yes, I jammed these in here for now to get the existing views to work with limited modification.